js/utils/loadTemplate.js

```diff
@@ -3,7 +3,11 @@ var fs = require('fs');
 
 module.exports = function(templateFile, callback){
 
-  fs.readFile(templateFile, "utf8", function(err, ob){
+  var loc = window.location.pathname;
+  var cur_dir = loc.substring(0, loc.lastIndexOf('/'));
+
+  fs.readFile(cur_dir + '/' + templateFile, "utf8", function(err, ob){
```

Comment:
looks like you should be using [`path.sep`](https://nodejs.org/api/path.html#path_path_sep) @hoffmabc 

Might also want to search for it on line 7.
```
var cur_dir = loc.substring(0, loc.lastIndexOf(path.sep));
```